### PR TITLE
Fix requirements for emoji plugin

### PR DESCRIPTION
**Pull request by FedericoCozziVM**
I encountered some issues installing this plugin, about unsatisfied conditions and requisites, so I thought to update readme and plugin with non-default packages

### CODE REVIEW

**Comment by FedericoCozziVM** [comment]
@bergercookie it's me or the failure of the build pipeline is related to other plugins, untouched by this PR?
```
============================= test session starts ==============================
platform linux -- Python 3.6.7, pytest-7.0.1, pluggy-0.12.0
rootdir: /home/travis/build/bergercookie/awesome-albert-plugins
collected 6 items                                                              
plugins/gmaps/gmaps-cli/tests/test_gmaps_cli.py ....s                    [ 83%]
plugins/saxophone/test/test_links.py F                                   [100%]
=================================== FAILURES ===================================
```
Could you check my code and eventually merge it anyway? I'm new to PRs on github, so I don't know if you can :smile:

File: plugins/emoji/README.md [hunk 1]
```diff
@@ -25,7 +25,30 @@ On additional key presses it allows for fuzzy search on the labels of each emoji
 
 ## Installation instructions
 
-Install [em-keyboard](https://github.com/hugovk/em-keyboard) via `pip3`:
+### Requisites
+
+#### Exec requisites
```
Comment (bergercookie) [inline plugins/emoji/README.md]: Maybe `Prerequisites` instead ?
Same for the rest of the usages of the term in the README

File: plugins/emoji/__init__.py [hunk 2]
```diff
@@ -13,14 +13,14 @@
 import pickle
 
 __title__ = "Emoji picker"
-__version__ = "0.4.0"
+__version__ = "0.4.1"
```
Comment (bergercookie) [inline plugins/emoji/__init__.py]: Let's revert this - this is the albert version this plugin depends on and currently all plugins in this repo use `0.4.0. Was this required for the plugin to work on your end?

File: plugins/emoji/README.md [hunk 3]
```diff
@@ -25,7 +25,30 @@ On additional key presses it allows for fuzzy search on the labels of each emoji
...
+
+- `xclip`: you must have installed xclip; ensure that xlip path is in PATH environment variable.
+
+_Example for Ubuntu_:
+
+```bash
+sudo apt-get install xclip
+which xclip  # i.e. /usr/bin/xclip
+export PATH="$PATH:/usr/bin/xclip"
```
Comment (bergercookie) [inline plugins/emoji/README.md]: For this to work you'd have to write

```
export PATH="$PATH:/usr/bin/" # <-- directory that xclip is in
```

But then again `/usr/bin` is always in your PATH on Ubuntu (and most other distros)

Also, if `which xclip` does find the executable, then it's most likely already in your PATH (see `man xclip` for more).

I'd skip the bash snippet in this case and just stick to:

> `xclip`: you must have installed xclip; ensure that xlip path is in PATH environment variable.`

Users will probably know how to modify PATH if required

**Review by bergercookie — CHANGES_REQUESTED** [review]

**Comment by bergercookie** [comment]
Hi @FedericoCozziVM , the CI failure is indeed irrelevant - don't worry about it.
Other than that left some minor comments - once addressed we can merge :)

File: plugins/emoji/__init__.py (hunk 2 above)
Comment (FedericoCozziVM) [reply plugins/emoji/__init__.py]: Sorry my bad, I thought that the version was specific for the plugin

File: plugins/emoji/README.md (hunk 3 above)
Comment (FedericoCozziVM) [reply plugins/emoji/README.md]: You're right, I'll change that

**Review by nikoskoukis-slamcore — APPROVED** [review]

**Review by bergercookie — APPROVED** [review]

**Comment by bergercookie** [comment]
Thanks!

**Pull request merged**